index.bs

```diff
@@ -138,8 +147,29 @@ intersection with the <a>intersection root</a>, as per the
 The IntersectionObserver interface</h3>
 
 The {{IntersectionObserver}} interface can be used to observe changes in the
-intersection of a <a>target</a> {{Element}} and a {{IntersectionObserver/root}}
-{{Element}} (or the top-level document's viewport).
+intersection of a {{IntersectionObserver/root}} {{Element}} (or the
+<a>top-level browsing context</a>'s visual viewport), and one or more <a>target</a> {{Element}}s.
+
+An {{IntersectionObserver}} with a {{IntersectionObserver/root}} {{Element}} can
+observe any <a>target</a> {{Element}} that is a descendant of the
+{{IntersectionObserver/root}} in the <a>node tree</a>, including {{Element}}s inside
+an <a>iframe</a> that belongs to the same <a>unit of related similar-origin browsing contexts</a>
```

Comment:
As per recent issue discussions, this text isn't right anymore, right? Namely:
1. Elements inside similar-origin browsing context frames are no longer allowed.
2. Root needs to be an ancestor of target in the containingBlock chain.